### PR TITLE
Fix for issue#50 - github container ignores HTTP errors

### PR DESCRIPTION
**Pull request by miggs125**
fixes issue#50

### CODE REVIEW

File: Github/service.js [hunk 1]
```diff
@@ -66,9 +66,9 @@ const getCommits = commitObject => {
         headers: { 'User-Agent': 'request' },
       },
       (err, res, data) => {
-        if (err) {
-          console.log('Error:', err);
-          reject(err);
+        if (res.statusCode !== 200) {
+          console.log('Error:', res.statusMessage);
+          reject(new Error(`Unable to get repos.`));
```
Comment (manekenpix) [inline Github/service.js]: nitpick: unnecessary bactkticks (same in line 106)

File: Github/service.js [hunk 2]
```diff
@@ -101,9 +101,9 @@ const getBranches = branchObject => {
         url: branchObject.url,
         headers: { 'User-Agent': 'request' },
       }, (err, res, data) => {
-        if (err) {
-          console.log('Error:', err);
-          reject(err);
+        if (res.statusCode !== 200) {
+          console.log('Error:', res.statusMessage);
+          reject(new Error(`Unable to get repos.`));
```
Comment (belavina) [inline Github/service.js]: nitpick: you are repeating `Unable to get repos.` msg in every function, would be useful to know if error happened in `get Commits`, `getRepos` or `getBranches` etc

Comment (miggs125) [reply Github/service.js]: fixed

File: Github/service.js (hunk 1 above)
Comment (miggs125) [reply Github/service.js]: fixed

**Pull request merged**